smartcontract/service/native/common.go

```diff
@@ -205,11 +231,35 @@ func getStorageBigInt(native *NativeService, key []byte) (*big.Int, error) {
 	return new(big.Int).SetBytes(item.Value), nil
 }
 
+func getStorageAdmin(native *NativeService, key []byte) (*states.Admin, error) {
```

Comment:
use contract address as param, and gen key in this function

Reply:
construct the transfer admin and origin admin key need different input parameter, not only contract address.

Reply:
Ah, I missed that.